.circleci/config.yml

```diff
@@ -60,7 +60,7 @@ commands:
       - run:
           name: Run tests
           working_directory: << parameters.directory >>
-          command: bundle exec fastlane test
+          command: bundle exec fastlane test wait_for_simulator:true
```

Comment:
added this param so that you don't wait for the simulator to start up when running tests locally, since our computers are much, much faster than the ones in CI